components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -175,7 +179,14 @@ - (void)viewWillLayoutSubviews {
   }
   CGSize size = [self.header sizeThatFits:CGRectStandardize(self.view.bounds).size];
   self.header.frame = CGRectMake(0, 0, self.view.bounds.size.width, size.height);
-  UIEdgeInsets insets = UIEdgeInsetsMake(self.header.frame.size.height, 0, 0, 0);
+  if (self.title != nil || self.message != nil) {
+    [self.view addSubview:self.headerDividerView];
```

Comment:
Won't this re-add the divider view on every layout pass? Can you check to see if the divider view's `superview` is `self.view`? Or maybe to the `addSubview:` and `removeFromSuperview` in the `setHeaderDividerColor:` method?

Reply:
This has been removed.